framework/src/modules/poa/constants.ts

```diff
@@ -43,3 +42,7 @@ export const defaultConfig = {
 export const KEY_SNAPSHOT_0 = utils.intToBuffer(0, 4);
 export const KEY_SNAPSHOT_1 = utils.intToBuffer(1, 4);
 export const KEY_SNAPSHOT_2 = utils.intToBuffer(2, 4);
+export const SUBSTORE_PREFIX_VALIDATOR = 0;
+export const SUBSTORE_PREFIX_CHAIN = 1;
+export const SUBSTORE_PREFIX_NAME = 2;
+export const SUBSTORE_PREFIX_SNAPSHOT = 3;
```

Comment:
can we rename them to `SUBSTORE_PREFIX_VALIDATOR_INDEX` as these values varies from the values given in the LIP but these indexes will be used to compute the given values in the LIP